Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,11 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+#root {
```

Comment:
I think it just comes down to preference since it's basically doing the same thing, but I would leave the root id alone and apply my styles else where.